src/Control/Monad/Catch.hs

```diff
@@ -448,11 +606,11 @@ onException action handler = action `catchAll` \e -> handler >> throwM e
 -- If an exception occurs during the use, the release still happens before the
 -- exception is rethrown.
```

Comment:
Put in the documentation for `bracket` a brief mention of `generalBracket`, as well how they differ.